lib/OpsGenieConnector.cpp

```diff
@@ -0,0 +1,28 @@
+#include "http_client.h"
+#include "OpsGenieConnector.h"
+
+const char *OpsGenieConnector::apiUrl = "https://api.opsgenie.com/v1/json/alert";
+
+OpsGenieConnector::OpsGenieConnector(const char *apiKey, const char *recipient) {
+    this->apiKey = apiKey;
+    this->recipient = recipient;
+}
+
+int OpsGenieConnector::pushAlert(const char *message) {
+    const char *json_format = "{\"message\":\"%s\", \"apiKey\": \"%s\", \"recipients\": [\"%s\"]}";
```

Comment:
Removed the recipients configuration. This will require using a separate integration(api key) for ProxySQL if it's default recipients differ from other integrations.